src/services/bundleAnalysis/usePagedBundleAssets.tsx

```diff
@@ -207,8 +207,8 @@ interface UsePagedBundleAssetsArgs {
     reportGroups?: string[]
     loadTypes?: string[]
   }
-  orderingDirection: 'ASC' | 'DESC'
-  ordering: 'NAME' | 'SIZE' | 'TYPE'
+  orderingDirection?: 'ASC' | 'DESC'
+  ordering?: 'NAME' | 'SIZE' | 'TYPE'
```

Comment:
It slightly annoys me that we don't have the ordering direction defined somewhere shared lol

Reply:
I'll fix in a follow up